doc/content/getting-started/next-steps/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Next steps"
+description: ""
+weight: 2
+---
+
+Once you have your first LoRaWAN Network setup, here are the next steps.
+
+<!--more-->
+
+1. [Connect a gateway]({{< ref "/gateways/concepts/adding-gateways" >}}) to your LoRaWAN Network.
+2. [Add LoRaWAN end devices]({{< ref "devices/adding-devices" >}}) to send LoRaWAN traffic to your network.
+3. [Create an integration]({{< ref "integrations" >}}) to send your LoRaWAN sensor data to external integrations or to your own private server.
+4. Learn how to get the most out of [{{% tts %}}]({{< ref "/the-things-stack/" >}}).
+5. Check out the [Glossary]({{< ref "/reference/glossary" >}}) for a list of commonly used terms.
+
+## Learn more
+
+LoRaWAN is a very deep topic that requires a lot of technical knowhow. Use the links suggested here to learn more.
+
+### LoRaWAN Crash Course
+
+Here is a video of Johan Stokking, the tech lead of The Things Industries and one of the core developers of {{% tts %}}, explaining LoRaWAN:
+
+{{< youtube "ZsVhYiX4_6o" >}}
+
+{{< note >}} Some common [use cases](https://www.thethingsnetwork.org/docs/lorawan/what-is-lorawan/#lorawan-use-cases) are explained in the video above, but there are certain limitations that make LoRaWAN not suitable for every use-case. Read about [LoRaWAN limitations in The Things Network LoRaWAN documentation](https://www.thethingsnetwork.org/docs/lorawan/limitations/). {{</ note >}}
+
+### Get Certified
+
+The Things Network certification program allows you to officially certify your LoRaWAN knowledge and skills.
+
+Visit [The Things Network Learn hub](https://www.thethingsnetwork.org/docs/lorawan/) or [The Things Academy on Udemy](https://www.udemy.com/course/lorawan-fundamentals/) to learn all about LoRa and LoRaWAN, and get prepared for [The Things Fundamentals certification](https://www.thethingsnetwork.org/achievements/a/the-things-certified-fundamentals/).
+
+Check all available certifications on [The Things Certification site](https://www.thethingsnetwork.org/achievements/certificates/).
+
+A lot of academic research around LoRaWAN is being conducted nowadays. [Read some LoRaWAN-related academic papers and publications](https://www.thethingsnetwork.org/docs/lorawan/academic/).
+
+### Useful links
+
+- Check out the [Learn section](https://thethingsnetwork.org/docs/lorawan) of The Things Network website.
+- Read the official specificationa at the [LoRa Alliance](https://lora-alliance.org).
```

Comment:
I would place this content together with the 'LoRaWAN' section in /getting-started, and maybe even make that a separate page ('LoRaWAN basics' or something similar). To me it feels like this doesn't belong here, and the existing /getting-started would be too packed if we just place it there.

Reply:
Ok I've added a "LoRaWAN Basics" section as the first guide.  This content here should belong in this `next-steps` section. We should not overwhelm new readers with all the details right at the start.